references/classification/utils.py

```diff
@@ -400,3 +401,57 @@ def reduce_across_processes(val):
     dist.barrier()
     dist.all_reduce(t)
     return t
+
+
+def set_weight_decay(
+    model: torch.nn.Module,
+    weight_decay: float,
+    norm_weight_decay: Optional[float] = None,
+    bias_weight_decay: Optional[float] = None,
+    custom_keys_weight_decay: Optional[Dict[str, float]] = None,
+):
+    norm_classes = (torch.nn.modules.batchnorm._BatchNorm, torch.nn.LayerNorm, torch.nn.GroupNorm)
```

Comment:
Might be worth adding `nn.modules.instancenorm._InstanceNorm` and `nn.LocalResponseNorm`. I've got the same update on my detection recipe [PR](https://github.com/pytorch/vision/pull/5444/files#diff-250a5139d0d5b18ff3513442f13dcb3760c9dfb1a8eeea2ecc3bf0f129dd220cR51).